Stack/readme.md

```diff
@@ -26,5 +26,7 @@ element in a full stack is called **overflow**.
 -   Evaluation of postfix expression ----> [C++](/Code/C++/Postfixexpression.cpp)
 -   Stack Class ----> [C++](/Code/C++/stack_class.cpp)
 -   Balanced Bracket Problem ----> [C++](/Code/C++/balanced_bracket.cpp)
+-   Stock Span Problem ----> [C++](/Code/C++/Stock_Span_Problem.cpp) | [Python](/Code/Python/StockSpan.py)
```

Comment:
I can see duplicate stock span problem please correct

Reply:
@plazzy99 I have corrected it..